types/mapbox__mapbox-sdk/index.d.ts

```diff
@@ -1329,6 +1329,10 @@ declare module '@mapbox/mapbox-sdk/services/static' {
     }
 
     interface PathOverlay {
+        path: Path;
```

Comment:
Right on, I see the jsdoc for [PathOverlay](https://github.com/mapbox/mapbox-sdk-js/blob/8ba742e04878b726d8f3967241961de836e2a0b5/services/static.js#L344) now. I originally snagged these types from [here](https://github.com/mapbox/mapbox-sdk-js/pull/294), there may be some other mistakes hiding out in these types.

I did see there was another PR out to generate types from the jsdoc too, fingers crossed for those types.